lib/game.ex

```diff
@@ -15,4 +16,29 @@ defmodule TicTacToe.Game do
     Map.replace!(game, :players, Game.get_players(input))
   end
 
+  def add_board(game) do
+    Map.replace!(game, :board, Board.new(3))
+  end
+
+  def make_move(game, move, input \\ CLI) do
+    [ current_player | _tail ] = game.players
+    cond do
+      !Board.space_is_available(game.board.spaces, move) ->
+        input.print "That space isn't available, please choose an open space between 1 and #{length(game.board.spaces)}"
+        new_move = input.get_player_move(current_player.name)
+        make_move(game, new_move, input)
+      Board.space_is_available(game.board.spaces, move) ->
```

Comment:
You seem to be referencing `game.board.spaces` a few times in this file, if it's super important, maybe there should be a shallower way to grab the info without going 2-levels down for it.

Reply:
I'm simplifying this by passing the board to any board module function (rather than the list of spaces), then having that function grab the spaces where it's needed